draft-irtf-cfrg-opaque.md

```diff
@@ -517,33 +517,24 @@ See {{validation}} for more details.
 
 ~~~
 struct {
-    opaque data<1..2^16-1>;
+    SerializedElement data;
 } RegistrationRequest;
 ~~~
 
 data
-: An encoded element in the OPRF group.
+: A serialized OPRF group element.
 
 ~~~
 struct {
-    opaque data_blind<1..2^16-1>;
-} RequestMetadata;
-~~~
-
-data_blind
-: An encoded OPRF scalar element.
-
-~~~
-struct {
-    opaque data<0..2^16-1>;
+    SerializedElement data;
```

Comment:
Where is SerializedElement defined? Perhaps a reference to an RFC #?

Reply:
I clarified that this is an alias for a serialized group element, i.e., output of `Serialize()`.